### PR TITLE
CNV-24682: archive policies

### PR DESCRIPTION
**Pull request by upalatucci**
Screenshot from 2023-01-24 11-29-11
image

Screenshot from 2023-01-24 11-02-53
!{Video](https://user-images.githubusercontent.com/29160323/214264861-3e5dc0ed-9395-4d65-8177-32eae8bb355d.mp4)

### CODE REVIEW

**Review by yaacov — APPROVED** [review]
lgtm

File: src/policies/details/PolicyPageTitle.tsx [hunk 1]
```diff
@@ -17,13 +20,17 @@ type PolicyPageTitleProps = {
...
+
+  const allEnactmentsAvailable = areAllEnactmentsAvailable(enactments);
+  const policyAbsent = areAllEnactmentsAbsent(enactments);
 
   return (
     <>
       <div className="pf-c-page__main-breadcrumb">
         <Breadcrumb className="pf-c-breadcrumb co-breadcrumb">
           <BreadcrumbItem>
-            <Link to={`/k8s/ns/cluster/${NodeNetworkConfigurationPolicyModelRef}`}>
```
Comment (yaacov) [inline src/policies/details/PolicyPageTitle.tsx]: could be fixed in a different pr ?

Comment (upalatucci) [reply src/policies/details/PolicyPageTitle.tsx]: yes okay

File: src/policies/components/ArchiveModal.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+              </ActionListItem>
+            </ActionList>
+          </StackItem>
+        </Stack>
+      }
+      isOpen={isOpen}
+      id="archive-modal"
+    >
+      <form id="archive-policy-form" onSubmit={handleSubmit}>
+        <p>{t('Are you sure you want to archive this policy?')}</p>
```
Comment (yaacov) [inline src/policies/components/ArchiveModal.tsx]:
```suggestion
        <p>{t('Archiving will remove the policy from all nodes, are you sure you want to archive this policy?')}</p>
```

**Review by yaacov — APPROVED** [review]
lgtm

**Pull request merged**